src/DynamoCoreWpf/ViewModels/Core/DynamoViewModel.cs

```diff
@@ -81,6 +82,22 @@ public partial class DynamoViewModel : ViewModelBase, IDynamoViewModel
         /// </summary>
         internal Dictionary<string, ViewExtensionDisplayMode> NodeWindowsState { get; set; } = new Dictionary<string, ViewExtensionDisplayMode>();
 
+        /// <summary>
+        /// Collection of Right SideBar tab items: view extensions and docked windows.
+        /// </summary>
+        public ObservableCollection<TabItem> SideBarTabItems
```

Comment:
Is `TabItem` type of WPF view elements? If yes, do you think we still want to store them under viewModel?

Reply:
It is a window control element. I moved it to DynamoViewModel so that I can bind it to the view and don't have to set the tabcontrol content everytime the collection is changed. I was seeing combox(for engine value) issues when we were not using binding and have to update it everytime.